### DISCUSSION TITLE
No response after github auth.

### DISCUSSION THREAD

**Discussion by gzrjzcx** [decaporg/decap-cms#7245] [Q&A]
Hi there, I am a newbie here and I have added both index.html and config.yml for my hexo blog projects, followed the official tutorial. As the results, I can see the Login with Github auth webpage poped up, and finished the auth process. However, after the auth process, there is nothing changed. I cannot enter the admin page like the official demo. And the auth window hasn't any button can close that page and step to the admin page. What should I do now? 

![image](https://github.com/decaporg/decap-cms/assets/33338567/dfae3a5d-d8c5-463b-aaab-46cca29e0538)

Thanks.

**Comment by rhyspowell** [comment]
I have exactly the same issue, just dropped a question in discord. If I get any assistance there, will post here but also dropping this comment incase we get an answer here

**Comment by moxplod** [comment]
Same issue - did you guys figure out how to solve this?

**Comment by tenhobi** [comment]
What is your config? I had similar issue when I didnt set `auth_endpoint` in `backend` in config.yaml

**Comment by adilarshidin** [comment]
Don't know if it is useful for you but dropping this for anyone else that searches this up.

The OAuth service needs to return the token to your Decap application and it should start logging in. The reason that doesn't happen is that you didn't define the callback url, like the commenter above mentioned.

You can check this by opening dev console in your Decap instance and watching the network tab.
